packages/hash/api/src/seed-data/dev-users.ts

```diff
@@ -43,13 +46,43 @@ export const ensureDevUsersAreSeeded = async ({
 }): Promise<UserModel[]> => {
   const createdUsers = [];
 
-  for (const { email, shortname, preferredName, isInstanceAdmin } of devUsers) {
+  // Only use `devUsers` if we are in a dev environment
+  let usersToSeed = !isDevEnv ? [] : devUsers;
+
+  // Or if we're explicitly setting users to seed.
+  if (process.env.HASH_SEED_USERS) {
+    try {
+      usersToSeed = JSON.parse(
+        process.env.HASH_SEED_USERS,
+      ) as DevelopmentUser[];
```

Comment:
Maybe down the line we could use a JSON-Schema to validate the structure of the JSON once it's been parsed, so the casting is a little safer here. If you agree, maybe worth leaving a `@todo` here for now.

Reply:
Added in 160061d6b173c9440ad9490aeb4b995666530823